heartbeat/sybaseASE

```diff
@@ -0,0 +1,895 @@
+#!/bin/bash
+
+#
+# Sybase Availability Agent for Red Hat Cluster v15.0.2
+# Copyright (C) - 2007
+# Sybase, Inc. All rights reserved.
+#
+# Sybase Availability Agent for Red Hat Cluster v15.0.2 is licensed
+# under the GNU General Public License Version 2.
+#
+# Author(s):
+#    Jian-ping Hui <jphui@sybase.com>
+#
+# Description: Service script for starting/stopping/monitoring \
+#              Sybase Adaptive Server on: \
+#                            Red Hat Enterprise Linux 5 ES \
+#                            Red Hat Enterprise Linux 5 AS
+#
+# NOTES:
+#
+# (1) Before running this script, we assume that user has installed
+#     Sybase ASE 15.0.2 or higher version on the machine. Please
+#     customize your configuration in /etc/cluster/cluster.conf according
+#     to your actual environment. We assume the following files exist before
+#     you start the service:
+#         /$sybase_home/SYBASE.sh
+#         /$sybase_home/$sybase_ase/install/RUN_$server_name
+#
+# (2) You can customize the interval value in the meta-data section if needed:
+#                <action name="start" timeout="300" />
+#                <action name="stop" timeout="300" />
+#
+#                <!-- Checks to see if it''s mounted in the right place -->
+#                <action name="status"  interval="30" timeout="100" />
+#                <action name="monitor" interval="30" timeout="100" />
+#
+#                <!--Checks to see if we can read from the mountpoint -->
+#                <action name="status" depth="10" timeout="100" interval="120" />
+#                <action name="monitor" depth="10" timeout="100" interval="120" />
+#
+#                <action name="meta-data" timeout="5" />
+#                <action name="validate-all" timeout="5" />
+#     The timeout value is not supported by Redhat in RHCS5.0.
+#
+# (3) This script should be put under /usr/share/cluster. Its owner should be "root" with
+#     execution permission.
```

Comment:
Is this really true, still?

Reply:
Oh. It was made for RGManager first, and then ported, so I'll remove that part :)